src/components/cart-icon/cart-icon.tsx

```diff
@@ -0,0 +1,20 @@
+import classNames from 'classnames';
+import styles from './cart-icon.module.scss';
+
+interface CartIconProps {
+    className?: string;
+    children: string;
+}
+
+export const CartIcon = (props: CartIconProps) => {
+    return (
+        <div className={classNames(styles.root, props.className)}>
```

Comment:
You don't need this div.

Reply:
I removed `div.cart-button` from your example and leave `div.root`. 
Do you mean `div.cart-button` is better then `div.root` for user using Codux? What about `props.className`? Do we allow users to add classes to our components? (Cart, Header, etc)

Reply:
Is this comment outdated and related to `CartIcon` based on `svg`?